packages/ui-components/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FunctionComponent } from "react";
+import classNames from "classnames/bind";
+
+import styles from "./Badge.module.scss";
+
+type BadgeCase = "none" | "uppercase";
+type BadgeIntent = "neutral" | "success" | "warning" | "danger";
+
+export interface BadgeProps {
+  content?: string;
```

Comment:
Is `content` an option for backwards compatibility in Admin UI? Having two ways to define text content opens an opportunity for incorrect usage/unexpected behavior:

```js
<Badge content="This won't be used">This will be used</Badge>
```

Reply:
Good point.